modules/installing-microshift-rpm.adoc

```diff
@@ -0,0 +1,28 @@
+= Installing MicroShift
+
+Perform the following steps to install MicroShift:
+
+1. Enable the MicroShift repositories:
+
+    sudo subscription-manager repos \
+        --enable rhocp-4.12-for-rhel-8-$(uname -i)-rpms \
+        --enable fast-datapath-for-rhel-8-$(uname -i)-rpms
+
+2. Install MicroShift:
+
+    sudo dnf install -y microshift
+
+3. Download your installation https://console.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat Hybrid Cloud Console]. This pull secret allows you to authenticate with the container registries that serve the container images used by MicroShift.
+
+4. Copy that pull secret to your RHEL machine at `/etc/crio/openshift-pull-secret` and make it read/write-able by the root user only:
+
+    sudo chown root. /etc/crio/openshift-pull-secret
+    sudo chmod 600 /etc/crio/openshift-pull-secret
+
+5. Configure at least the mandatory firewall rules (see https://xxx[firewall configuration] for more information). If your RHEL machine uses `firewalld` (the default):
+
+    sudo firewall-cmd --permanent --zone=trusted --add-source=10.42.0.0/16
+    sudo firewall-cmd --permanent --zone=trusted --add-source=169.254.169.1
+    sudo firewall-cmd --reload
```

Comment:
This assumes a running firewall. We may want to use offline commands here.
For the reload, we can check if the service is active.

Reply:
As I wrote above, we can assume `firewalld` to be running in most cases and the offline commands do not correctly work if it _is_ running.

IDK how the docs team typically handles this, but I believe we can assume an audience that has pretty good Linux skills and keep the instructions as simple and clear as possible. In the rare cases that users uninstalled or disabled the firewall, they should be able do the right thing.